manager/dispatcher/dispatcher.go

```diff
@@ -66,8 +67,18 @@ var (
 	ErrSessionInvalid = errors.New("session invalid")
 	// ErrNodeNotFound returned when the Node doesn't exist in raft.
 	ErrNodeNotFound = errors.New("node not found")
+
+	// Scheduling delay timer.
+	schedulingDelayTimer metrics.Timer
 )
 
+func init() {
+	ns := metrics.NewNamespace("swarm", "dispatcher", nil)
+	schedulingDelayTimer = ns.NewTimer("scheduling_delay",
+		"Scheduling delay, which is basically (task_status_running - task_status_creation).")
```

Comment:
I don't understand the last part of this.

Maybe "Scheduling delay, which is the time a task spends between the NEW and RUNNING states".